platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -32,7 +32,7 @@
         <AD description="List of authentication types required for each context. List of default valid authentication types are: SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
```

Comment:
❗️ Description here should be updated to include `IDP` as a valid authentication type.